server/worker/jira.go

```diff
@@ -1,31 +1,137 @@
 package worker
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"fmt"
+	"html/template"
 
+	jira "github.com/andygrunwald/go-jira"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 )
 
-type Jira struct {
-	ds  fleet.Datastore
-	log kitlog.Logger
-	// TODO: add jira client
+const (
+	// JiraName is the name of the job as registered in the worker.
+	JiraName = "jira"
+
+	nvdCVEURL = "https://nvd.nist.gov/vuln/detail/"
+)
+
+var jiraSummaryTmpl = template.Must(template.New("").Parse(
+	`{{ .CVE }} detected on hosts`,
+))
+
+// TODO: check if jira api supports markdown in summary/description
+var jiraDescriptionTmpl = template.Must(template.New("").Parse(
```

Comment:
This (and making sure creating issues works with just the ProjectKey, without the project ID) is to be tested against a real Jira instance - unclear in the Jira API docs.

Reply:
I can confirm that it can create issues with just the project key, and we don't need markdown for our uses.